package.json

```diff
@@ -34,30 +34,30 @@
         "dist/"
     ],
     "scripts": {
+        "preinstall": "npx only-allow pnpm",
         "build": "rimraf dist && tsc -p ./tsconfig.build.json",
         "contributors:add": "all-contributors add",
         "contributors:generate": "all-contributors generate",
         "format": "prettier --write '**/*.{js,md,json,yaml,yml}'",
         "lint": "eslint --fix '**/*.ts'",
         "prepare": "husky install",
-        "test": "jest",
-        "test:coverage": "jest --coverage",
-        "test:watch": "jest --watch"
+        "test": "vitest run",
+        "test:coverage": "vitest run --coverage",
+        "test:watch": "vitest"
     },
     "devDependencies": {
         "@tool-belt/eslint-config": "^1.3.3",
-        "@types/jest": "^29.2.4",
         "@types/node": "^18.11.11",
+        "@vitest/coverage-istanbul": "^0.34.2",
         "all-contributors-cli": "^6.24.0",
         "eslint": "^8.29.0",
         "husky": "^8.0.2",
-        "jest": "^29.3.1",
         "lint-staged": "^13.1.0",
         "prettier": "^2.8.1",
         "rimraf": "^3.0.2",
-        "ts-jest": "^29.0.3",
         "ts-node": "^10.9.1",
-        "typescript": "^4.9.3"
+        "typescript": "^4.9.5",
```

Comment:
I think you can run `pnpm up --latest` - all should be ok.

Reply:
yes, i tried something else it braking, one problem at a time.